SwiftReflector/MachOHelpers.cs

```diff
@@ -52,6 +52,9 @@ static string ToArchString (MachO.Architectures arch)
 				return "i386";
 			case MachO.Architectures.x86_64:
 				return "x86_64";
+			// TJ - adding for arm64e support
```

Comment:
Is this valid?

Reply:
yes and no
it's valid for an MachO file
but it's not yet supported by mono (or, less sure, coreclr)